test/Utils.h

```diff
@@ -143,12 +143,12 @@ OscProb::PMNS_NUNM* GetNUNM(bool is_nominal){
   OscProb::PMNS_NUNM* p = new OscProb::PMNS_NUNM(0);
   SetNominalPars(p);
   if(!is_nominal){
-    p->SetAlpha(0,0, 0.1, 0);
-    p->SetAlpha(1,0, 0.2, 0);
-    p->SetAlpha(2,0, 0.3, 0);
-    p->SetAlpha(1,1, 0.4, 0);
-    p->SetAlpha(2,1, 0.5, 0);
-    p->SetAlpha(2,2, 0.6, 0);
+    p->SetAlpha(0,0, 0.05, 0);
+    p->SetAlpha(1,0, 0.06, 0);
+    p->SetAlpha(2,0, 0.07, 0);
+    p->SetAlpha(1,1, 0.08, 0);
+    p->SetAlpha(2,1, 0.09, 0);
+    p->SetAlpha(2,2, 0.1, 0);
```

Comment:
Why did you change the test values here? It's not a big problem, but if you kept the same values you could have kept the same ROOT test file to compare that probabilities are identical. Did you check that the values were indeed the same before you changed this?

Reply:
I have changed them to match the values used to test the sterile model. I do not expect them to be identical to before because the new implementation changes the way the probabilities is propagated, this recovers consistency with sterile when also the off diag pars are non zero.

Reply:
Alright, so this means we have a bug fix in this pull request. Duly noted. And indeed we shouldn't expect consistency with the previous version.